packages/dashboard/src/vaadin-dashboard.d.ts

```diff
@@ -217,7 +218,7 @@ export interface DashboardI18n {
  * @fires {CustomEvent} dashboard-item-resize-mode-changed - Fired when an item resize mode changed
  */
 declare class Dashboard<TItem extends DashboardItem = DashboardItem> extends DashboardLayoutMixin(
-  ElementMixin(ThemableMixin(HTMLElement)),
+  I18nMixin({} as DashboardI18n, ElementMixin(ThemableMixin(HTMLElement))),
```

Comment:
With the way the `Dashboard` class is defined, applying the I18nMixin required some creativity.